pkg/controller/daemon/daemon_controller.go

```diff
@@ -218,6 +218,8 @@ func NewDaemonSetsController(
 	dsc.podStoreSynced = podInformer.Informer().HasSynced
 	controller.AddPodNodeNameIndexer(podInformer.Informer())
 	dsc.podIndexer = podInformer.Informer().GetIndexer()
+	controller.AddPodControllerUIDIndexer(podInformer.Informer())
+	dsc.podIndexer = podInformer.Informer().GetIndexer()
```

Comment:
Setting `podIndexer` twice doesn't make much sense. Please change to:
```
controller.AddPodNodeNameIndexer(podInformer.Informer())
controller.AddPodControllerUIDIndexer(podInformer.Informer())
dsc.podIndexer = podInformer.Informer().GetIndexer()
```

Reply:
Ah, from the merge from other PR, didn't realize. Like your eye for detail :D